.github/workflows/trigger-build.yml

```diff
@@ -1,18 +1,16 @@
-name: Trigger master website deploy
+name: Trigger website deploy
 on:
   push:
     branches:
```

Comment:
We could scope down the file paths that trigger this, but I decided to leave this as is for now. The rate of change on the documentation repository isn't that high.